crates/solidity/outputs/cargo/tests/src/bindings_assertions/runner.rs

```diff
@@ -0,0 +1,52 @@
+use std::fs;
+
+use anyhow::Result;
+use infra_utils::cargo::CargoWorkspace;
+use semver::Version;
+use slang_solidity::bindings;
+use slang_solidity::language::Language;
+
+use super::assertions::{check_assertions, collect_assertions};
+use super::generated::VERSION_BREAKS;
+
+pub fn run(group_name: &str, test_name: &str) -> Result<()> {
+    let file_name = format!("{test_name}.sol");
+    let data_dir = CargoWorkspace::locate_source_crate("solidity_testing_snapshots")?
+        .join("bindings_assertions")
+        .join(group_name);
+    let input_path = data_dir.join(file_name);
+    let input = fs::read_to_string(&input_path)?;
+
+    for version in &VERSION_BREAKS {
+        check_assertions_with_version(version, input_path.to_str().unwrap(), &input)?;
+    }
+    Ok(())
+}
+
+fn check_assertions_with_version(
+    version: &Version,
+    file_path: &str,
+    file_contents: &str,
+) -> Result<()> {
+    let language = Language::new(version.clone())?;
+
+    let parse_output = language.parse(Language::ROOT_KIND, file_contents);
+    if !parse_output.is_valid() {
+        eprintln!("WARNING: Parsing failed for {file_path} with version {version}");
```

Comment:
I suggest printing the errors directly instead of this warning, so that readers can correlate any failed bindings to the parse errors directly.